_episodes/08-testing.md

```diff
@@ -463,10 +460,10 @@ The test will pass if the `build_bond_list` method raises a `ValueError`, otherw
 
 ## Exercise - Test Driven Development
 
-Sometimes, tests are written before code is actually written. 
+It is common practice in software development to write tests before actually writing the code. 
```

Comment:
I'd like this to be "sometimes" rather than stating it's common practice. Can it be changed to be closer to the original?